simplified-books-database/src/main/java/org/nypl/simplified/books/book_database/BookDRMInformationHandleACS.kt

```diff
@@ -0,0 +1,200 @@
+package org.nypl.simplified.books.book_database
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.nypl.drm.core.AdobeAdeptLoan
+import org.nypl.drm.core.AdobeLoanID
+import org.nypl.simplified.books.api.BookDRMInformation
+import org.nypl.simplified.books.api.BookDRMKind
+import org.nypl.simplified.books.book_database.api.BookDRMInformationHandle
+import org.nypl.simplified.books.book_database.api.BookFormats
+import org.nypl.simplified.files.FileUtilities
+import org.nypl.simplified.json.core.JSONParserUtilities
+import org.nypl.simplified.json.core.JSONSerializerUtilities
+import java.io.ByteArrayOutputStream
+import java.io.File
+import java.io.IOException
+import java.nio.ByteBuffer
+import java.util.concurrent.atomic.AtomicBoolean
+
+/**
+ * An information handle for Adobe ACS DRM.
+ */
+
+class BookDRMInformationHandleACS(
+  private val directory: File,
+  format: BookFormats.BookFormatDefinition,
+  private val onUpdate: () -> Unit
+) : BookDRMInformationHandle.ACSHandle(), BookDRMInformationHandleBase {
+
+  private val objectMapper = ObjectMapper()
+  private val closed = AtomicBoolean(false)
+
+  companion object {
+    fun nameRights(format: BookFormats.BookFormatDefinition): String =
+      "${format.shortName}-rights_adobe.xml"
+
+    fun nameACSM(format: BookFormats.BookFormatDefinition): String =
+      "${format.shortName}-meta_adobe.acsm"
+
+    fun nameMetaJSON(format: BookFormats.BookFormatDefinition) =
+      "${format.shortName}-meta_adobe.json"
+
+    fun names(format: BookFormats.BookFormatDefinition) =
+      listOf(
+        nameRights(format),
+        nameACSM(format),
+        nameMetaJSON(format)
+      )
+  }
+
+  init {
+    BookDRMInformationHandles.writeDRMInfo(
+      directory = this.directory,
+      format = format,
+      kind = BookDRMKind.ACS
+    )
+  }
+
+  private val fileAdobeRights: File =
```

Comment:
The explicit value type could be removed since it can be easily inferred for all these file handles.

Mostly just looking for some kind of consistency here.